Covid/UI/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  Covid
+//
+//  Created by Boris Bielik on 20/04/2020.
+//  Copyright © 2020 Sygic. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ViewController: UIViewController {
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.isTranslucent = true
```

Comment:
Toto moze byt kludne sucastou napr. UIAppearance, ale zatial som to tymto smerom nepushoval